332 - Fix the modal was disappears if its content use checkbox in it with rtl (#2263)
infor-design/enterprise: docs/CHANGELOG.md

### v4.20.0 Fixes

- `[Homepages]` Fixed an issue where personalize and chart text colors were not working with hero. ([#2097](https://github.com/infor-design/enterprise/issues/2097))

> - `[Modal]` Fixed an issue where Modal was disappears if its content use checkbox in it with RTL settings. ([#332](https://github.com/infor-design/enterprise-ng/issues/332))

Reviewer: ```suggestion
- `[Modal]` Fixed an issue where the modal component would disappears if its content had a checkbox in it in RTL. ([#332](https://github.com/infor-design/enterprise-ng/issues/332))
```

> - `[Modal]` Fixed an issue where the modal component would disappears if its content had a checkbox in it in RTL. ([#332](https://github.com/infor-design/enterprise-ng/issues/332))

Reviewer 2: ```suggestion
- `[Modal]` Fixed an issue where the modal component would disappear if its content had a checkbox in it in RTL. ([#332](https://github.com/infor-design/enterprise-ng/issues/332))
```
